wifiphisher/pywifiphisher.py

```diff
@@ -310,6 +340,18 @@ def start(self):
         # setup the logging configuration
         setup_logging(args)
 
+        if args.data_path:
+            if args.data_path[-1] != os.path.sep:
```

Comment:
It's not clear what this check does. Can you please add a comment above this line?